util/extsort/disk_sorter.go

```diff
@@ -80,37 +746,123 @@ func OpenDiskSorter(dirname string, opts *DiskSorterOptions) (*DiskSorter, error
 	}
 	opts.ensureDefaults()
 
-	dbOpts := &pebble.Options{
-		MaxConcurrentCompactions: opts.Concurrency,
-		DisableWAL:               true,
-		L0CompactionThreshold:    math.MaxInt,
-		L0StopWritesThreshold:    math.MaxInt,
+	fs := vfs.Default
+	if dirname == "" {
+		fs = vfs.NewMem()
+	}
+	if err := fs.MkdirAll(dirname, 0755); err != nil {
```

Comment:
not sure what's the effect of an empty dirname 🤔

Reply:
It does nothing when dirname is empty.